spec/system/admins/company_values_spec.rb

```diff
@@ -0,0 +1,39 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
+
+RSpec.describe 'Kudos' do
+  let(:admin) { create(:admin) }
+
+  before do
+    sign_in admin
+  end
+
+  it 'All CRUD Company Values' do
```

Comment:
I suggest splitting this test into separate actions, so you have one test per each action - something like:
it 'creates new Company Value' and so on

also it looks like Read/Show test is missing ;)

Reply:
I've added READ/Show - good point, I have missed it! :)

I created all CRUD actions in one IT - because I wanted not to overload rspec test and not to create list of Company Values but to base on the only first one which I have created and check if they are on the page or not and the same with edit/destroy actions. 

I hope you get my point! 
But.. I have additionally added more tests to check if it's not flash-message but that it was also changed in the record. 

And I have added this part to check if the user is admin or not.

Reply:
I get your point :)

As regards non-admin access I suggest adding test of a non-validated user visiting /admins path - because right now you have only checking if the link is not visible :)